src/__tests__/generateConfluenceMarkdown.test.ts

```diff
@@ -44,8 +44,8 @@ describe('generateConfluenceMarkdown function', () => {
 
         const oneDependencyNoIssue = issuesByDependency.set(dependency, noIssues);
         const results = mariner.generateConfluenceMarkdown(oneDependencyNoIssue);
-        expect(results).not.toContain(`h3. ${dependency}`);
-        expect(results).not.toContain('\n ||*Title*||*Age*||');
+        expect(results).not.toContain(dependency);
+        expect(results).not.toContain('||*Title*||*Age*||');
         expect(results).not.toContain('|days|');
```

Comment:
I think this should be 'days', because there would always be at least a space between the pipe and the d.